exporter/fileexporter/README.md

```diff
@@ -20,16 +23,29 @@ This intended for primarily for debugging Collector without setting up backends.
 The following settings are required:
 
 - `path` (no default): where to write information.
+- `rolling_logger_options` (default): an option to self-rotate log files.
+    - max_size:   the maximum size in megabytes of the log file before it gets rotated. It defaults to 100 megabytes.
```

Comment:
How about using strings and parsing them as byte sizes, like with https://pkg.go.dev/github.com/alecthomas/units?

Reply:
I had the same thought, but the underlying library does not allow it: https://github.com/natefinch/lumberjack/issues/37

It looks like it may be added in the next major version, so perhaps then we can implement `max_size` as you've suggested (and still support `max_megabytes` if necessary).